crates/mfem/examples/ex1.rs

```diff
@@ -96,33 +80,32 @@ fn main() -> anyhow::Result<()> {
     //    the boundary attributes from the mesh as essential (Dirichlet) and
     //    converting them to a list of true dofs.
     let mut ess_tdof_list = ArrayInt::new();
-    if let Some(max_bdr_attr) = mesh.get_bdr_attributes().iter().max() {
+    if let Some(max_bdr_attr) = fespace.mesh().bdr_attributes().iter().max() {
         let mut ess_bdr = ArrayInt::with_len(*max_bdr_attr as usize);
-        ess_bdr.set_all(1);
+        ess_bdr.fill(1);
```

Comment:
Good rename!

Reply:
Actually, it is the method coming from `&[i32]` to which `ArrayInt` [deref](https://github.com/Chris00/mfem-rs/blob/autocxx/crates/mfem/src/lib.rs#L315)s.